Formula/hhvm-nightly.rb

```diff
@@ -37,6 +37,9 @@ class HhvmNightly < Formula
   depends_on "pkg-config" => :build
   depends_on "re2c" => :build
   depends_on "wget" => :build
+  
+  # Provides the CLI utility `realpath`
+  depends_on "aardvark_shell_utils" => :build
```

Comment:
GNU coreutils (`"coreutils"`) is the usual source for this, and what's used on linux (so more compatible) - any reason for preferring this alternative implementation?

Reply:
Good question. Why do you use `md5sha1sum` instead of `coreutils`?

https://github.com/hhvm/homebrew-hhvm/commit/c887f08864f4aa9634d383f00a1deb417633c953

Reply:
From internal diff, due to https://github.com/hhvm/homebrew-hhvm/commit/c887f08864f4aa9634d383f00a1deb417633c953

The `coreutils` package can conflict with other packages providing identically named executables (e.g. it would also conflict with aarvark_shell_utils, which may also conflict with other stuff)

Let's try coreutils instead, and see if there are still problems.

Reply:
The md5/shasum stuff was largely resolved by moving to `openssl dgst`

Reply:
As discussed in private chat. We will use `aardvark_shell_utils` temporarily to fix the build. Then we will clean up the `md5sha1sum` and `aardvark_shell_utils` dependencies to solve the conflicts and hopefully finally switch to `coreutils`.